app/vep/models/pipeline_model.py

```diff
@@ -17,5 +20,19 @@ class LaunchParams(BaseModel):
     labelIds: List[str]
     headJobCpus: int = 1
     headJobMemoryMb: int = 4
+
 class PipelineParams(BaseModel):
     launch: LaunchParams
+
+class ConfigIniParams(BaseModel):
+  cache: int = 1
+  dir_cache: str = "/nfs/production/flicek/ensembl/variation/data/VEP/tabixconverted/"
```

Comment:
Instead of hardcoding this can also be exposed through env variable.

Reply:
> Instead of hardcoding this can also be exposed through env variable.

This is not something static and permanent though; it will eventually be a dynamic parameter that comes from the metadata api. Why put it in an env variable?

Reply:
yeah it should come from metadata api which is why I didnt bother to change.